website/docs/05_design/dependency-management.md

```diff
@@ -12,38 +12,67 @@ The following decision tree states how the Package Operator is handling dependen
   the components interacting with the user (CLI, UI) need to guide them through potential resolution. Consequently, the only time the operator does resolve an unfulfilled
   dependency, the "result" is denoted as `install`.
 
+<div style={{maxWidth: '100%', overflow: 'auto'}}>
+<div id="dep-diagram" style={{width: '1600px'}}>
+
+```mermaid
+%%{init: {'themeVariables': {'flowchart': {'useWidth': 10000} }}}%%
+flowchart LR
+
+Start --> A("Does P require a<br>version range of D?")
+
+%% Branch when P requires no version range of D
+
+A -->|No| B("Does D exist?")
+
+B -->|Yes| C("Are there other packages<br>dependent on D?")
+
+C -->|No| Fulfilled["P -> D is fulfilled"]
+
+C -->|Yes| E("Do X and Y require<br>no version range of D?")
+
+E ---->|Yes| Fulfilled
+
+E ---->|No| Fulfilled
```

Comment:
I just had a chat with @christophenne and we came to the conclusion that this branch is unnecessarily complex and should be simplified
```suggestion
B -->|Yes| Fulfilled["P -> D is fulfilled"]
```